README.md

```diff
@@ -16,6 +16,40 @@ For details about configuration and deployment of RBD and CephFS CSI plugins, se
 
 For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
 
+## Support Matrix
+
+### Ceph-CSI features and available versions
+
+|   Plugin |        Features                                           | CSI driver Version |
+|----------|-----------------------------------------------------------|--------------------|
+|   CephFS | Dynamically provision, de-provision File mode RWO volume  |      >=v0.3.0      |
+|          | Dynamically provision, de-provision File mode RWX volume  |      >=v0.3.0      |
+|          | Creating and deleting snapshot                            |          -         |
+|          | Provision volume from snapshot                            |          -         |
+|          | Provision volume from another volume                      |          -         |
+|          | Resize volume                                             |          -         |
+|          |                                                           |                    |
+|   RBD    | Dynamically provision, de-provision Block mode RWO volume |      >=v0.3.0      |
+|          | Dynamically provision, de-provision Block mode RWX volume |      >=v0.3.0      |
+|          | Dynamically provision, de-provision File mode RWO volume  |        v1.0.0      |
+|          | Creating and deleting snapshot                            |      >=v0.3.0      |
+|          | Provision volume from snapshot                            |        v1.0.0      |
+|          | Provision volume from another volume                      |          -         |
+|          | Resize volume                                             |          -         |
+
+### Ceph-CSI versions and CSI spec compatibility
+
+| Ceph CSI driver Version | CSI spec version |
+|-------------------------|------------------|
+|         v0.3.0          |     v0.3         |
+|         v1.0.0          |     v1.0.0       |
+|          master         |     v1.1.0       |
```

Comment:
we have only having v0.3.0 and v1.0.0 driver versions
as this is a driver version, not the branch name we need to remove the master.